react/features/stream-effects/blur/JitsiStreamBlurEffect.js

```diff
@@ -35,10 +33,13 @@ export default class JitsiStreamBlurEffect {
      * Represents a modified video MediaStream track.
      *
      * @class
-     * @param {BodyPix} bpModel - BodyPix model.
+     * @param {Object} bpModel - Meet model.
+     * @param {Object} dimensionOptions - Segmentation dimensions.
      */
-    constructor(bpModel: Object) {
+    constructor(bpModel: Object, dimensionOptions: Object) {
```

Comment:
Let's call it `options`, since we might add more things than dimension, such as the blur factor, later on.